### PR TITLE
feat: documenting alias_card filter

### PR DESCRIPTION
**Pull request by jleon15**
## Description

- Document alias_card filter

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: docs/expressions/filters.mdx [hunk 1]
```diff
@@ -68,10 +68,11 @@ Given a token with the data:
 
 | Expression                                            | Result                  |
 | ----------------------------------------------------- | ----------------------- |
-| <code>{{ data &#124; card_mask: 'false', 'false', 'X', '-' }}</code>              | `"XXXX-XXXX-XXXX-XXXX"` |
-| <code>{{ data &#124; card_mask: 'true', 'false', 'X', '-' }}</code>              | `"3622-7206-XXXX-XXXX"` |
-| <code>{{ data &#124; card_mask: 'true', 'true', '#', '-' }}</code>              | `"3622-7206-####-5567"` |
-| <code>{{ data &#124; card_mask: 'false', 'true', 'X', '-' }}</code>              | `"XXXX-XXXX-XXXX-5567"` |
+| <code>{{ data &#124; alias_card }}</code>              | `"3292-2312-4920-2291"` |
```
Comment (jleon15) [inline docs/expressions/filters.mdx:71]: i wonder if we should use `card_number` type instead as that's what i would recommend if tokenizing card numbers, but then we cannot demonstrate the examples preserving the delimiters as only generic tokens can contain spaces or dashes in the card number

Comment (djejaquino) [reply docs/expressions/filters.mdx:71]: Yes. We should demonstrate usage with both `card` (`{{data.number | alias_card }}`)  and `card_number`

File: docs/expressions/filters.mdx [hunk 2]
```diff
@@ -142,6 +143,40 @@ Given a token with the data:
 | <code>{{ data &#124; alias_preserve_length: 0, 5 }}</code> | `"v38anr9m2cx67890"` |
 | <code>{{ data &#124; alias_preserve_length: 3, 5 }}</code> | `"ABCanr9m2cx67890"` |
 
+## card_mask
```
Comment (djejaquino) [inline docs/expressions/filters.mdx:146]: Same thing for this filter.

**Review by djejaquino — CHANGES_REQUESTED** [review]

File: docs/expressions/filters.mdx (hunk 1 above)
Comment (jleon15) [reply docs/expressions/filters.mdx:71]: right now the example is using token type `token`, I wouldn't want to add 3 examples for each, so maybe just 1 more, either `card` or `card_number`

**Pull request merged**